staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/celcoststability_test.go

```diff
@@ -384,7 +384,7 @@ func TestCelCostStability(t *testing.T) {
 
 				// two variable comprehensions
 				"self.val.all(k, v, v > 0)":                                             13,
-				"self.val.exists(k, v, v == 2)":                                         15,
+				"!self.val.exists(k, v, v == 3)":                                        16,
```

Comment:
add a comment that this *has* to not match any items so it fully iterates over the randomly ordered map (instead of short-circuiting) and is deterministic